kn/fotos/entities.py

```diff
@@ -71,7 +72,7 @@ def id(self):
     def required_visibility(self, user):
         if user is None:
             return frozenset(('world',))
-        if 'webcie' in user.cached_groups_names:
+        if user == 'system' or 'webcie' in user.cached_groups_names:
```

Comment:
`system` is een toegestane gebruikersnaam: dat is dus onhandig.  Het is beter `!system` te gebruiken – die gebruikersnaam is niet toegestaan.